src/composables/useGame.ts

```diff
@@ -86,6 +90,16 @@ export function useGame() {
       // replace the winner coins with the double value to highlight them
       remarkWinner(winner);
 
+      // displaies confetties after a victory
+      setTimeout(() => {
+        displayConfetti();
+      }, 1000);
+
+      // removes canvas element after confetties animation
+      setTimeout(() => {
+        removeConfettiFromBody();
+      }, 2000);
+
```

Comment:
Replace by this:
```
confetti({
  particleCount: 100,
  spread: 200,
  origin: { y: 0.5 },
});
```

Reply:
@ByMykel 
The reason I did it this way is to avoid two things:
- first: to ensure that the animation will appear only after the cirlce reaches its spot; that's why I used a setTimeout function;
- second: once the canvas is inside the `body` element its necessary removes it from `body`... otherwise the canvas element will block the `RESTART` button, for example;

Also, I wanted to isolate all the logic to generate the confetti animation on one function.

If I did what you're suggested the confetti functionaliy will not work properly. Can you run the code I wrote on your machine to see if what I'm saying make any sence? Tell me if you had the same problems I did when implementing the solution.

Reply:
What I'm trying to say is that you don't need `displayConfetti` and `removeConfettiFromBody` because `confetti({...})` already does that for you.

About the first point you right, but you still don't need all the extra code you wrote.

I've tested your code an it works correctly what I think is overcomplicated.

Reply:
@ByMykel 
Got it! Thank you.

Reply:
@dinobergviana Can you open a new PR and make all this cleaner?

Reply:
@ByMykel 
Yes! After I refact it I'm going to open a new PR.